services/backend/src/docs/definitions.yml

```diff
@@ -24,6 +24,12 @@ definitions:
       - C
       - E
       - X
+  CardVisibilityStatusEnum:
+    type: string
+    enum: string
```

Comment:
```suggestion
    enum:
```

Or else shows this instead of one of the values of the enum
![image](https://user-images.githubusercontent.com/22248828/89666824-0731d380-d8a9-11ea-82fa-caf30e77aad2.png)